SpatialGDK/Documentation/content/get-started/example-project/exampleproject-local-deployment.md

```diff
@@ -8,7 +8,7 @@ When you want to try out your game, you need to launch a deployment; the deploym
 There are two types of deployment: local and cloud.
 
 - A **local deployment** is for testing only. A local deployment runs your game simulation in your Unreal Editor. 
-</br>Your game simulation with its own instance of the SpatialOS Runtime, runs on your development machine. You can run multiple clients in a local deployment - they are useful for fast development iteration.
+</br>Your game simulation with its own instance of the SpatialOS Runtime, runs on your development machine. You can run multiple clients in a local deployment. This type of deployment is useful for fast development iteration.
```

Comment:
I'm still uneasy about these sentences, but not your changes. Why the comma in the first sentence?

Reply:
"Your game simulation with its own instance of the SpatialOS Runtime runs on your development machine. You can run multiple clients in a local deployment, which is useful for fast development iteration."

May be a better option?

Reply:
Agree with Jo on this. Either we need to add a comma after "game simulation", or remove the comma that's currently there. I'd probably go for adding a comma, as otherwise the sentence is a little hard to parse.

I also like Jo's simplification of the second and third sentences into one.

So I'd recommend:

> Your game simulation, with its own instance of the SpatialOS Runtime, runs on your development machine. You can run multiple clients in a local deployment, which is useful for fast development iteration.